src/pywatemsedem/__init__.py

```diff
@@ -10,8 +10,33 @@
     del version, PackageNotFoundError
 
 # check if SAGA/WaTEM topology is installed properly
+import os
+import shutil
 import subprocess
+from pathlib import Path
 
+# Check if saga_cmd executable can be found
+if shutil.which("saga_cmd") is None:  # saga_cmd cannot be found in the PATH variable
+    # Check if there is an environment variable "SAGA"
+    if os.environ.get("SAGA") is not None and Path(os.environ.get("SAGA")).exists():
+        os.environ["SAGA_TLB"] = os.environ.get("SAGA")
```

Comment:
I'm a bit suprised this is the same folder. Do I understand correctly you then add the dll files in the folder with saga, and not under tools